### ISSUE TITLE
Build is slow, semantic-release is failing post-deployment

### ISSUE THREAD

**Issue by trieloff** [adobe/helix-embed#171]
See https://app.circleci.com/pipelines/github/adobe/helix-embed/712/workflows/5f616b76-bed5-4e0d-9115-1354d673ff70:

the `build` step takes 10 minutes, the `semantic-release` step takes 31 minutes and is failing.

**Comment by tripodsan** [comment]
I tried to work around the hanging tests... and it most of the times works, but sometimes the tests still hang. It is related to using helix-fetch, which keeps the connections open. alternatively, we could use `--exit` on the mocha tests for force stopping the process.

**Comment by trieloff** [comment]
Ok, I'll watch the CI build and close this issue if resolved.

**Comment by tripodsan** [comment]
ok, I tried to merge....

I also notived, that the tests only hang, if something fails:

```
[ERROR] Status 403: request failed, check your url
```

**Comment by tripodsan** [comment]
no....that's not it.... now it went through

**Comment by trieloff** [comment]
Yep. I'm glad that we are dealing with all the HTTP2 paper cuts now, instead of later when we have lots of traffic.

**Comment by tripodsan** [comment]
> I'm glad that we are dealing with all the HTTP2 paper cuts now,

I don't think it's related to http2, but to helix-fetch :-)

@stefan-guggisberg would it be possible to add an option to helix-fetch to disable keep-alive ?

**Comment by stefan-guggisberg** [comment]
@tripodsan 

```js
const { fetch } = require('@adobe/helix-fetch').context({
  http1: {
    keepAlive: false
  },
  httpsProtocols: ['http1']
});
```

see [here](https://github.com/grantila/fetch-h2#keep-alive)

**Comment by MarquiseRosier** [comment]
^^ Hey @stefan-guggisberg that is what I do with the proxyquire. I figured it would cured us of the hanging; but that was merged yesterday. All the fetching is done with keepAlive set to false!

**Comment by stefan-guggisberg** [comment]
I have reported a related [fetch-h2 issue](https://github.com/grantila/fetch-h2/issues/77) a while ago. It's been finally fixed 2 days ago and the fix is included in `helix-fetch` [v1.4.1](https://github.com/adobe/helix-fetch/releases/tag/v1.4.1) released yesterday. 

Disabling `keep-alive` shouldn't be necessary.  I will verify this in another PR.
